conf/parsers/yaml_parser.py

```diff
@@ -5,8 +5,16 @@
 
 
 def parse(file_stream):
-    # Parse the given file stream of yaml format to a dict.
-    loaded = yaml.load(file_stream, Loader=yaml.FullLoader)
+    """
+    Parse the given file stream of yaml format to a dict.
+    """
+    major, minor = [int(i) for i in yaml.__version__.split('.')][0:2]
+
+    if (major == 5 and minor >= 1) or (major > 5):
```

Comment:
You could also just check if ``FullLoader`` exists, rather than checking the version. Line 11 could then be removed.

```suggestion
    if hasattr(yaml, 'FullLoader'):
```